samples/wifi/provisioning/sample_description.rst

```diff
@@ -0,0 +1,95 @@
+.. _wifi_provisioning_sample_desc:
+
+Sample description
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Wi-Fi Provisioning Service sample demonstrates how to provision a device with Nordic Semiconductor's Wi-Fi chipsets over Bluetooth® Low Energy.
+
+.. _wifi_provisioning_app:
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+The sample requires a smartphone (configurator) with Nordic Semiconductor's nRF Wi-Fi Provisioner app installed in one of the following versions:
+
+* `nRF Wi-Fi Provisioner mobile app for Android`_
+* `nRF Wi-Fi Provisioner mobile app for iOS`_
+
+Overview
+********
+
+With this sample, you can provision a Wi-Fi device that lacks input or output capability.
+The sample is divided into three parts:
+
+* Task and event handling component: Handles provisioning-related tasks and events.
+* Transport layer: Based on Bluetooth Low Energy and exchanges information between the device and the nRF Wi-Fi Provisioner app.
+* Configuration management component: Manages the provisioning data (in RAM and flash) accessed by multiple threads.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/provisioning`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+|test_sample|
+
+1. |connect_kit|
+#. |connect_terminal|
+#. Observe the following output:
+
+    .. code-block:: console
+
+       wifi_prov: BT Advertising successfully started
+
+#. Open the nRF Wi-Fi Provisioner app.
+#. Enable Bluetooth if it is disabled.
+#. Search and connect to the device you want to provision. If the connection and pairing are established, observe the following output:
+
+    .. code-block:: console
+
+       wifi_prov: BT Connected
+       wifi_prov: BT Pairing Complete
+
+#. Conduct the access point (AP) scan, enter password if required, and trigger Wi-Fi connection.
+#. Observe the following output, if the connection is successful:
+
+    .. code-block:: console
+
+       wpa_supp: wlan0: CTRL-EVENT-CONNECTED - Connection to <AP MAC Address> completed [id=0 id_str=]
+
+Dependencies
+************
+
+This sample uses the following `sdk-nrfxlib`_ library:
+
+* :ref:`nrfxlib:nrf_security`
+
+This sample also uses modules that can be found in the following locations
+in the |NCS| folder structure:
+
+* ``modules/lib/nanopb``
```

Comment:
```suggestion
* :file:`modules/lib/nanopb`
```
apply proper filepath formatting to this and header files in lines 88-91, 95.

Reply:
For the dependency section, the style is copied from other README.rst files. Not sure if we should change it.